### PR TITLE
Restructured repo and brought tf examples

### PR DESCRIPTION
**Pull request by retr0h**

### CODE REVIEW

**Comment by retr0h** [comment]
May not be 100% exact but I believe this structure is workable.

File: README.md [hunk 1]
```diff
@@ -1,52 +1,27 @@
-# Corelight Cloud
+# Corelight Deployment Guide
 
-IaC used to deploy Corelight Sensors into various Cloud Providers.
+This repository provides sample configurations for deploying Corelight products
+across AWS, Azure, and Google Cloud Platform (GCP). The examples offer both
+**Terraform-based** and **cloud-native IaC solutions**, allowing users to
```
Comment (thathaneydude) [inline README.md]:
```suggestion
**Terraform-based** and **cloud-native IaC solutions**, allowing you to
```

File: terraform/aws-cloud-enrichment/README.md [hunk 2]
```diff
@@ -0,0 +1,11 @@
+# Corelight Cloud Enrichment Service Deployment - AWS
+
+This directory provides Terraform code for deploying Corelight's Cloud Enrichment
+on **GCP**.
```
Comment (thathaneydude) [inline terraform/aws-cloud-enrichment/README.md]:
```suggestion
on **AWS**.
```

File: terraform/azure-cloud-enrichment/README.md [hunk 3]
```diff
@@ -0,0 +1,11 @@
+# Corelight Cloud Enrichment Service Deployment - Azure
+
+This directory provides Terraform code for deploying Corelight's Cloud Enrichment
+on **GCP**.
```
Comment (thathaneydude) [inline terraform/azure-cloud-enrichment/README.md]:
```suggestion
on **Azure**.
```

**Review by thathaneydude — CHANGES_REQUESTED** [review]
A couple minor things but I like it!

**Review by thathaneydude — APPROVED** [review]

**Pull request merged**